docs/release-notes.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_label: Release Notes
+title: ''
+---
+
+# Release Notes
+
+The Elemental project stack is made of various components such as the `Operator` and `UI` for example.
+
+Each of these components have an independent development lifecycle with its own versioning. Once a new version is ready, meaning it fully integrates with the others components of the Elemental project stack, a release is made.
+
+Here's the different components, their latest version and a link to the respective release notes on GitHub:
+
+| Name                                                                 | Version | Release Notes                                                              |
+|----------------------------------------------------------------------|---------|----------------------------------------------------------------------------|
+| [Elemental Operator](https://github.com/rancher/elemental-operator/) | v1.2.5  | [Link](https://github.com/rancher/elemental-operator/releases/tag/v1.2.5)  |
+| [Elemental Toolkit](https://github.com/rancher/elemental-toolkit/)   | v0.10.7 | [Link](https://github.com/rancher/elemental-toolkit/releases/tag/v0.10.7)  |
+| [Elemental Linux](https://github.com/rancher/elemental)              | v1.1.5  | [Link](https://github.com/rancher/elemental/releases/tag/v1.1.5)           |
+| [Elemental UI](https://github.com/rancher/elemental-ui)              | v0.1.0  | [Link](https://github.com/rancher/elemental-ui/releases/tag/desktop-0.1.0) |
+
+:::note Information on docs versioning
+
+The docs versioning is based on `Elemental Operator` instead of `Elemental Linux`. The reason is that, as a user, the component used as an "entrypoint" to the Elemental project stack is the operator and not the Linux OS directly.
```

Comment:
🤔 the framing here looks to me that the right versioning should be based on Elemental Linux. Why not just state where the overall versioning comes from? Something like:
```suggestion
The docs versioning is based on the `Elemental Operator` component as it is the user "entrypoint" to the Elemental project stack.
```